src/Foundatio/Caching/HybridCacheClient.cs

```diff
@@ -87,15 +87,17 @@ public class HybridCacheClient : IHybridCacheClient {
         public async Task<int> RemoveAllAsync(IEnumerable<string> keys = null) {
             var items = keys?.ToArray();
             bool flushAll = items == null || items.Length == 0;
-            await _messageBus.PublishAsync(new InvalidateCache { CacheId = _cacheId, FlushAll = flushAll, Keys = items }).AnyContext();
             await _localCache.RemoveAllAsync(items).AnyContext();
-            return await _distributedCache.RemoveAllAsync(items).AnyContext();
+            var removed =  await _distributedCache.RemoveAllAsync(items).AnyContext();
```

Comment:
I was just giving this one last final review before merging in. Wouldn't we want to invalidate the remote cache first here? Otherwise you have a dirty read and possibly dirty set in the local cache. The same for RemoveByPrefix